tests/inspect/inspect.go

```diff
@@ -58,6 +62,26 @@ func main() {
 		fmt.Fprintf(os.Stdout, "%s=%s\n", globalFlags.PrintEnv, os.Getenv(globalFlags.PrintEnv))
 	}
 
+	if globalFlags.PrintCapsPid >= 0 {
+		const linuxCapabilityVersion3 = 0x20080522 // as documented in capget(2)
+		hdrp := []uint32{linuxCapabilityVersion3, uint32(globalFlags.PrintCapsPid)}
+		datap := []uint32{0, 0, 0, 0, 0, 0, 0}
+		_, _, errno := syscall.Syscall(syscall.SYS_CAPGET, uintptr(unsafe.Pointer(&hdrp[0])), uintptr(unsafe.Pointer(&datap[0])), 0)
```

Comment:
Consider filing a bug to `go` about implementing capget. Also, I read somewhere that syscall package is deprecated in favor of some sys/unix package (`import "golang.org/x/sys/unix"`).